docs/en/configuration/sink-plugins/Kudu.md

```diff
@@ -0,0 +1,31 @@
+# Sink plugin: Kudu
+
+## Description
+
+Write data to Kudu.
+
+## Options
+
+| name           | type   | required | default value |
+| -------------- | ------ | -------- | ------------- |
+| [kudu_master](#kudu_master-string)            | string | yes      | -             |
+| [kudu_table](#kudu_table-string)       | string | yes      | -         |
+| [mode](#mode-string)       | string | no      | insert         |
```

Comment:
Please remove `(#kudu_master-string)` , it doesn’t seem to be able to jump in markdown

Reply:
It works, you can test here : https://github.com/simon824/seatunnel/blob/kudu-sink/docs/en/configuration/sink-plugins/Kudu.md#kudu_master-string